GithubBrowserSample/app/build.gradle

```diff
@@ -104,7 +106,8 @@ dependencies {
 
     androidTestImplementation("android.arch.core:core-testing:$arch_version", {
     })
-    androidTestImplementation "org.mockito:mockito-android:$mockito_version"
+    androidTestImplementation "com.linkedin.dexmaker:dexmaker-mockito:$dexmaker_version", { exclude group: 'net.bytebuddy' }
```

Comment:
why is this needed?

Reply:
the arch testing incremented version in between that comes w/ this byte buddy problem.
(see build files of arch components for details). I think it is fixed in later versions but didn't want to try that.